openrpc.json

```diff
@@ -1,8569 +1,7685 @@
+--> Generating OpenRPC spec
```

Comment:
This will cause problems, can you update the Makefile to remove this echo?

Reply:
removed